index.js

```diff
@@ -75,53 +79,73 @@ const fixJson = (err, data, verbose) => {
   const targetLine = start.line - 2
 
   if (extraChar(err)) {
-    fixedData = fixer.fixExtraChar({fixedData, verbose, targetLine})
+    fixedData = fixer.fixExtraChar({ fixedData, verbose, targetLine })
   } else if (trailingChar(err)) {
-    fixedData = fixer.fixTrailingChar({start, fixedData, verbose})
+    fixedData = fixer.fixTrailingChar({ start, fixedData, verbose })
   } else if (missingChar(err)) {
     if (verbose) psw(chalk.magenta('Missing character'))
     const brokenLine = removeLinebreak(lines[targetLine])
     fixedData[targetLine] = `${brokenLine},`
   } else if (singleQuotes(err)) {
-    fixedData = fixer.fixSingleQuotes({start, fixedData, verbose})
+    fixedData = fixer.fixSingleQuotes({ start, fixedData, verbose })
   } else if (missingQuotes(err)) {
-    fixedData = fixer.fixMissingQuotes({start, fixedData, verbose})
+    fixedData = fixer.fixMissingQuotes({ start, fixedData, verbose })
   } else if (notSquare(err)) {
-    fixedData = fixer.fixSquareBrackets({start, fixedData, verbose, targetLine})
+    fixedData = fixer.fixSquareBrackets({ start, fixedData, verbose, targetLine })
   } else if (notCurly(err)) {
-    fixedData = fixer.fixCurlyBrackets({fixedData, verbose, targetLine})
+    fixedData = fixer.fixCurlyBrackets({ fixedData, verbose, targetLine })
   } else if (comment(err)) {
-    fixedData = fixer.fixComment({start, fixedData, verbose})
+    fixedData = fixer.fixComment({ start, fixedData, verbose })
   } else if (ops(err)) {
-    fixedData = fixer.fixOpConcat({start, fixedData, verbose})
+    fixedData = fixer.fixOpConcat({ start, fixedData, verbose })
   } else
     throw new Error(
       `Unsupported issue: ${err.message} (please open an issue at the repo)`,
     )
-  return doubleCheck(fixedData.join('\n'), verbose)
+  return doubleCheck(fixedData.join('\n'), options)
 }
 /*eslint-enable no-console */
 
 /**
  * @param {string} data JSON string data to check (and fix).
+ * @param {{verbose:boolean, parse:boolean}} options configuration object which specifies verbosity and whether the object should be parsed or returned as fixed string
  * @param {boolean} [verbose=false] Verbosity
  * @returns {{data: (Object|string|Array), changed: boolean}} Result
  */
-const checkJson = (data, verbose = false) => {
+const checkJson = (data, options, verbose = false) => {
```

Comment:
As `options` is there, do you think it would make more sense in merging `verbose` into it (ie. it will be a key in `options` like `parse`)?

That way, `checkJson` doesn't need an extraneous 3rd argument which could simply be specified in the second one (e.g. `checkJson(..., {parse: ..., verbose: ...})`.

Also, thank you for your contribution :grin:.